devices/saswell.js

```diff
@@ -15,6 +15,7 @@ module.exports = [
             {modelID: 'TS0601', manufacturerName: '_TZE200_c88teujp'},
             {modelID: 'TS0601', manufacturerName: '_TZE200_yw7cahqs'},
             {modelId: 'TS0601', manufacturerName: '_TZE200_azqp6ssj'},
+            {modelID: 'TS0601', manufacturerName: '_TYST11_zuhszj9s'},
```

Comment:
@drzony Isn't this typo in the end :hushed: ? In my log, I see `_TZE200_zuhszj9s`, not `_TYST11_zuhszj9s`.

Reply:
tried here maybe? https://github.com/Koenkk/zigbee-herdsman-converters/pull/2601

Reply:
Yes, that's the problem. Copy-Paste error on my side

Reply:
Thanks, really appreciate the help and the pointers :+1: